docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: "3.8"
```

Comment:
this file shouldn't be tracked .. it should be generated with the script that generate it from the template